### PR TITLE
Add PLD Linux

### PR DESCRIPTION
**Pull request by mistoo**
PLD (https://pld-linux.org/) LXD/LXC image. Tested with bin/build-distro script, but I'm not 100% sure about Jenkins job.

### CODE REVIEW

File: images/pld.yaml [hunk 1]
```diff
@@ -0,0 +1,146 @@
+image:
+  distribution: pld
+
+source:
+  downloader: docker-http
+  url: pldlinux/i686:latest
```
Comment (stgraber) [inline images/pld.yaml]: The `/i686` here would indicate that you're only ever pulling the 32bit image.
If there's a 64bit image, then hopefully just omitting the architecture will do the right thing.

If there's only a 32bit image available, then you shouldn't list amd64 as an architecture in the jenkins job.

**Comment by stgraber** [comment]
We're going to have to test it before merging as I'm not sure that Docker can be used in our image build environment. It would be the first image making use of this, so this may be a problem.

File: images/pld.yaml (hunk 1 above)
Comment (mistoo) [reply images/pld.yaml]: Yep, this is placeholder actually, as source.url is replaced by -o in jenkins job; as far as I know  distrobuilder is not able to handle template inlining here ({{ architecture }})

Comment (stgraber) [reply images/pld.yaml]: Ah, right, in this case it's probably best not to set it at all so we don't get confused again when reading it :)

Comment (mistoo) [reply images/pld.yaml]: Other readers may have no clue what is the image source then, maybe I just comment out this?

Comment (stgraber) [reply images/pld.yaml]: Yeah, I guess you can comment it out.

For other images, they're almost all lacking the `image.release` property, so folks using the YAMLs without knowing to feed the missing stuff will have issues already :)

**Pull request merged**